pyproject.toml

```
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "smashbox"
+version = "0.1.0"
+description = "Main test suite for CERNBox"
+authors = ["Your Name <you@example.com>"]
```

You can go ahead with:

```suggestion
authors = ["CERNBox Administrators <cernbox-admins@cern.ch>"]
```